images/s6_assets/pulp_tests.sh

```diff
@@ -21,6 +21,7 @@ else
   cd pulp-cli
 fi
 pip install -r test_requirements.txt || pip install --no-build-isolation -r test_requirements.txt
```

Comment:
Is this script meant to be run inside the container? Let's not do that.
By calling pip, we may end up with an incompatible version of pulp-glue for pulpcore.

Instead, I suggest: Install the cli, retrieve it's version and checkout the corresponding release tag to grant access to the matching test suite.

Reply:
@mdellweg No it's not. It is run on the host.